vagrant_support/bashrc

```diff
@@ -0,0 +1,11 @@
+# .bashrc
+
+# Source global definitions
+if [ -f /etc/bashrc ]; then
+    . /etc/bashrc
+fi
+
+# User specific aliases and functions
+PS1='\[\e[0;31m\]\u\[\e[m\] \[\e[1;34m\]\w\[\e[m\] $ '
```

Comment:
It was just to make it pretty, as a result I added a devtools.yml that can be used to inject a few other goodies later.